manifests/server/puppetserver.pp

```diff
@@ -41,13 +41,22 @@
   $jvm_max_heap_size = $::puppet::server_jvm_max_heap_size,
   $jvm_extra_args    = $::puppet::server_jvm_extra_args,
 ) {
+  
+  File {
+    notify => Class['puppet::server::service'],
+  }
+
+  $puppetserver_config_dir = versioncmp($::puppetversion, '4.0.0') >= 0 ? {
```

Comment:
@brandonweeks that reads like it shouldn't work. I think it's not needed in params.pp because there it's an if statement and not a selector. In any case, I think we should move this to params.pp and possibly reuse ```$aio_package```.